corpora/csip/metadata/mets-root/CSIP5/testCase.xml

```diff
@@ -1,23 +1,88 @@
+      The value of the other attribute determines the existence of this attribute.
+    </dependency>
+    <dependency requirementId="VocabularyContentInformationTypeSpecification" URL="https://earkcsip.dilcis.eu/schema/CSIPVocabularyContentInformationType.xml">
+      The value of the attribute MUST NOT be from the vocabulary.
+    </dependency>
   </dependencies>
-  <!-- A list of the validation rules derived from the test case -->
   <rules>
+    <!--  The following rules are included in the test case for CSIP4 and not repeated here:
+          - "If mets/@csip:CONTENTINFORMATIONTYPE = 'OTHER' then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist."
+          - "If mets/@csip:CONTENTINFORMATIONTYPE = 'OTHER' then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist and have a value."-->
+    <!-- This rule is implicit in the specification and is important for CSIP4 and CSIP5 to work as intended. -->
+    <rule id="1">
+      <description>If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST NOT have a value from the VocabularyContentInformationTypeSpecification because these vocabulary values MUST be used for the attribute mets/@csip:CONTENTINFORMATIONTYPE.</description>
+      <error level="ERROR">
+        <message>
+          mets/@csip:OTHERCONTENTINFORMATIONTYPE attribute has a value that MUST instead be used for mets/@csip:CONTENTINFORMATIONTYPE. MUST be: mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST describe a content information type that falls outside of the VocabularyContentInformationTypeSpecification vocabulary.
+        </message>
+      </error>
+      <corpusPackages>
+        <!-- Ideally, there should be an invalid IP for each specific vocabulary
+          value, as this is the only certain way to test that the validator
+          checks for all of them. This should be automated as
+          VocabularyContentInformationTypeSpecification currently allows 7 different values. -->
+        <package name="mets-xml_mets_csip-CONTENTINFORMATIONTYPE_OTHER_and_csip-OTHERCONTENTINFORMATIONTYPE_vocabulary_1" isValid="FALSE">
+          <path></path>
+          <description>
+            METS.xml file has mets/@csip:CONTENTINFORMATIONTYPE = "OTHER", mets/@csip:OTHERCONTENTINFORMATIONTYPE = "ERMS".
+          </description>
+        </package>
+        <package name="minimal_IP_with_1_rep_mets_csip-CONTENTINFORMATIONTYPE_OTHER" isValid="TRUE">
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements, mets/@csip:CONTENTINFORMATIONTYPE = "OTHER", mets/@csip:OTHERCONTENTINFORMATIONTYPE = "SIARDUK".
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <!-- This rule is implicit in the specification and is important for CSIP4 and CSIP5 to work as intended. -->
+    <rule id="2">
+      <description>If mets/@csip:OTHERCONTENTINFORMATIONTYPE exists then mets/@csip:CONTENTINFORMATIONTYPE MUST be "OTHER".</description>
```

Comment:
Or maybe also "MIXED"?

Reply:
I see the concern but I don't have enough experience to know if it's an issue in real life. If it is, we should propose a change to the A2 team. At the moment the CSIP5 description mentions only "OTHER," so I left it unchanged.